apps/monitor-client/lib/components/layout/ListItems.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import PropTypes from 'prop-types';
+
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import ListSubheader from '@mui/material/ListSubheader';
+import ViewListIcon from '@mui/icons-material/ViewList';
+import PeopleIcon from '@mui/icons-material/People';
+import WebhookIcon from '@mui/icons-material/Webhook';
+import AssignmentIcon from '@mui/icons-material/Assignment';
+
+export function MainListItems() {
```

Comment:
move to lib/components/

Reply:
moved to /lib/components/layout/sidebar/